### PR TITLE
create demo page testing #130 

### PR DESCRIPTION
**Pull request by rafaover**
All request was executed.
In the instructions was said: "export this component under client/src/pages/index.js as well. Follow the same format as you see inside that file."

I believe you wanted on client/src/index.js, right?

Thanks

### CODE REVIEW

File: client/src/pages/Demo.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Demo = () => {
```
Comment (tsimian) [inline client/src/pages/Demo.js:3]: Do we need this Demo component if we have the one in the Demo folder?

**Review by tsimian — COMMENTED** [review]
This looks good to me @lugenx, I just had a single question left in the comment. Great work @rafaover!

File: client/src/pages/Demo.js (hunk 1 above)
Comment (lugenx) [reply client/src/pages/Demo.js:3]: sorry, looks like i overlooked this question. i don't think so, i think we just need one demo page

Comment (rafaover) [reply client/src/pages/Demo.js:3]: Do I need to fix anything on this PR? Just following up

Comment (lugenx) [reply client/src/pages/Demo.js:3]: This looks good to me, thank you @rafaover

**Review by lugenx — APPROVED** [review]

**Pull request merged**